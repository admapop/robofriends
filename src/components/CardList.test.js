import { shallow } from 'enzyme'
import React from 'react'
import CardList from './CardList'

it('expect to render Card comp', () => {
    const mockRobots = [
        {
            id: 1,
            name: 'Josh Now',
            username: 'Josh',
            email: 'josh@gmail.com'
        }
    ]
    const cardListComponent = shallow(<CardList robots={mockRobots} />)
    expect(cardListComponent.length).toEqual(1)
    expect(cardListComponent.debug()).toMatchSnapshot()
})