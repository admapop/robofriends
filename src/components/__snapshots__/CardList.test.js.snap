// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Card comp 1`] = `
"<div>
  <Card id={1} name=\\"Josh Now\\" email=\\"josh@gmail.com\\" />
</div>"
`;
